_episodes/39.md

```diff
@@ -0,0 +1,273 @@
+     time: 4435
+---
+
+## Guests
+
+In this episode we will be talking to two engineers from 
+[Raft](https://goraft.tech/) who will discuss how they use Trino to connect data
+silos that exist across different departments in various government sectors:
+
+* [Edward Morgan](https://www.linkedin.com/in/edwardwmorgan/), 
+Senior Platform Engineer/DevSecOps Manager at Raft
+* [Steve Morgan](https://www.linkedin.com/in/steve-morgan-b9bb6642/), Chief
+Data Engineer at Raft
+
+## Register for Trino Summit 2022!
+
+Trino Summit 2022 is coming around the corner! It will be at the Commonwealth
+Club in San Francisco, CA on November 10th!  If you have a talk, the 
+[call for speakers](https://sessionize.com/trino-summit-2022/) is open until
+September 15th. 
+
+[Register for the conference](https://www.starburst.io/info/trinosummit/) at
+any time. We must limit in-person registrations to 250 
+attendees, so register soon if you plan on attending in person!
+
+## Releases 392 to 393
+
+Official highlights from Martin Traverso:
+
+[Trino 392](https://trino.io/docs/current/release/release-392.html)
+
+* Support for dynamic filtering with fault-tolerant query execution.
+* Support for correlated subqueries in `DELETE` queries.
+* Support for Amazon S3 Select pushdown for JSON files.
+* Support for Avro format in Iceberg connector.
+* Faster queries when filtering by `__time` column in Druid.
+
+[Trino 393](https://trino.io/docs/current/release/release-393.html)
+
+* Add support for `MERGE`.
+* Improved performance of highly selective `LIMIT` queries.
+* Experimental docker image for `ppc64le`.
+* Dynamic filtering support for various connectors.
+* Support for JSON and bytes type in Pinot.
+
+Additional highlights worth a mention according to Manfred:
+
+* Lots of other improvements on Delta Lake, Hive, and Iceberg connectors.
+* Merge support in a bunch of connectors.
+* OAuth 2.0 refresh token fixes
+
+More detailed information is available in the release notes for
+[Trino 392](https://trino.io/docs/current/release/release-392.html),
+and
+[Trino 393](https://trino.io/docs/current/release/release-393.html).
+
+## Concept of the episode: Trino at Raft
+
+Raft provides consulting services and is particularly skilled at DevSecOps. One
+particular challenge they face is dealing with fragmented government
+infrastructure. In this episode, we dive in to learn how Trino enables Raft to
+supply government sector clients with a data fabric solution. Raft takes a
+special stance on using and contributing to open source solutions that run well
+on the cloud.
+
+### Intro Software Factories
+
+A “software factory” is an organized approach to software development that
+provides software design and development teams a repeatable, well-defined path
+to create and update software. It results in a robust, compliant, and more
+resilient process for delivering applications to production.
+([Source](https://tanzu.vmware.com/software-factory))
```

Comment:
Linking to a source like this seems a little strange. Is vmware.com an authority, or are we paraphrasing directly from them or something? Might flow better to say "according to VMWare" or something along those lines so we don't have a dangling (Source) at the end of the paragraph.

Reply:
Fair, yeah we're directly quoting from them. VMWare owns Pivotal which has a large amount of these government contracts. They are somewhat of an authority in this space it seems.